source/calendar/calendar-script.js

```diff
@@ -196,3 +196,9 @@ document.addEventListener("DOMContentLoaded", function () {
     } 
   }
 });
+document.getElementById("search-bar").addEventListener("click", function() {
```

Comment:
try running prettier here, might be using single quotes. I can catch it later too